### ISSUE TITLE
HackRF Jawbreaker not working well

### ISSUE THREAD

**Issue by natevw** [gqrx-sdr/gqrx#146]
If I set bandwidth to 20 000 000, demod is too choppy to work but the spectogram/waterfall seems stable. If I lower bandwidth to 8 000 000 then the demod seems to work much better, **but** only for a few seconds — then it just stops receiving any samples or something (the interface isn't frozen, but no data is coming through) until I fully restart the app (using the on/off button doesn't help). Then it works for another ~2 seconds and goes silent again :-(

Basically any sampling speed below 20 000 000 makes the sampling "freeze up" soon after it starts, but my computer doesn't seem to be able to handle the full rate. Any idea why it won't work with lower sample rates?

This is OS X 10.9.1 on a 1.86GHz C2D laptop.
